# Análisis de Datos sobre Felicidad en Paises

## ¿Cómo acceden a los datos de la felicidad?

Como se trata de datos numéricos y strings, es recomendable leer esta información a través de pandas en vez de un numpy array.

In [5]:
import pandas as pd

In [6]:
url = 'https://raw.githubusercontent.com/cienciadedatos/datos-de-miercoles/master/datos/2019/2019-08-07/felicidad.csv'

In [7]:
felicidad =  pd.read_csv(url)

vemos que en efecto se leyó correctamente la información

In [8]:
felicidad.head(4)

pais  anio  escalera_vida   log_pib  soporte_social  \
0  Afghanistán  2008       3.723590  7.168690        0.450662   
1  Afghanistán  2009       4.401778  7.333790        0.552308   
2  Afghanistán  2010       4.758381  7.386629        0.539075   
3  Afghanistán  2011       3.831719  7.415019        0.521104   

   expectativa_vida  libertad  generosidad  percepcion_corrupcion  \
0         50.799999  0.718114     0.177889               0.881686   
1         51.200001  0.678896     0.200178               0.850035   
2         51.599998  0.600127     0.134353               0.706766   
3         51.919998  0.495901     0.172137               0.731109   

   afecto_positivo  afecto_negativo  confianza  calidad_democracia  \
0         0.517637         0.258195   0.612072           -1.929690   
1         0.583926         0.237092   0.611545           -2.044093   
2         0.618265         0.275324   0.299357           -1.991810   
3         0.611387         0.267175   0.307386           -1.919018   

   calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
0        -1.655084               1.774662                 NaN   
1        -1.635025               1.722688                 NaN   
2        -1.617176               1.878622                 NaN   
3        -1.616221               1.785360                 NaN   

   gini_banco_mundial_promedio  
0                          NaN  
1                          NaN  
2                          NaN  
3                          NaN

In [11]:
felicidad.tail(4)

pais  anio  escalera_vida   log_pib  soporte_social  \
1700  Zimbabue  2015       3.703191  7.556052        0.735800   
1701  Zimbabue  2016       3.735400  7.538829        0.768425   
1702  Zimbabue  2017       3.638300  7.549491        0.754147   
1703  Zimbabue  2018       3.616480  7.553395        0.775388   

      expectativa_vida  libertad  generosidad  percepcion_corrupcion  \
1700         53.799999  0.667193    -0.097354               0.810457   
1701         54.400002  0.732971    -0.068105               0.723612   
1702         55.000000  0.752826    -0.069670               0.751208   
1703         55.599998  0.762675    -0.038384               0.844209   

      afecto_positivo  afecto_negativo  confianza  calidad_democracia  \
1700         0.715079         0.178861   0.590012           -0.893078   
1701         0.737636         0.208555   0.699344           -0.900649   
1702         0.806428         0.224051   0.682647           -0.988153   
1703         0.710119         0.211726   0.550508                 NaN   

      calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
1700        -1.357514               2.198865                 NaN   
1701        -1.374650               2.776363                 NaN   
1702        -1.350867               2.656848                 NaN   
1703              NaN               2.498696                 NaN   

      gini_banco_mundial_promedio  
1700                        0.432  
1701                        0.432  
1702                        0.432  
1703                        0.432

## ¿Cómo calculan la medida de tendencia central y dispersión de la felicidad a nivel global para el primer y último año incluido en el set de datos?

vemos los valores únicos de los años presentes en el data set

In [9]:
felicidad["anio"].unique

<bound method Series.unique of 0       2008
1       2009
2       2010
3       2011
4       2012
        ... 
1699    2014
1700    2015
1701    2016
1702    2017
1703    2018
Name: anio, Length: 1704, dtype: int64>

como vemos que el primer año es 2009 y el último año es 2018

In [15]:
felicidad_anio = felicidad[(felicidad["anio"]==2009) | (felicidad["anio"]==2018)]
felicidad_anio

pais  anio  escalera_vida   log_pib  soporte_social  \
1     Afghanistán  2009       4.401778  7.333790        0.552308   
10    Afghanistán  2018       2.694303  7.494588        0.507516   
12        Albania  2009       5.485470  9.161638        0.833047   
21        Albania  2018       5.004403  9.412399        0.683592   
28        Algeria  2018       5.043086  9.557952        0.798651   
...           ...   ...            ...       ...             ...   
1678        Yemen  2018       3.057514       NaN        0.789422   
1682       Zambia  2009       5.260361  8.026193        0.781926   
1690       Zambia  2018       4.041488  8.223958        0.717720   
1694     Zimbabue  2009       4.055914  7.197595        0.805781   
1703     Zimbabue  2018       3.616480  7.553395        0.775388   

      expectativa_vida  libertad  generosidad  percepcion_corrupcion  \
1            51.200001  0.678896     0.200178               0.850035   
10           52.599998  0.373536    -0.084888               0.927606   
12           66.199997  0.525223    -0.160855               0.863665   
21           68.699997  0.824212     0.005385               0.899129   
28           65.900002  0.583381    -0.172413               0.758704   
...                ...       ...          ...                    ...   
1678         56.700001  0.552726          NaN               0.792587   
1682         48.639999  0.696183    -0.106159               0.916553   
1690         55.299999  0.790626     0.036644               0.810731   
1694         45.419998  0.411089    -0.055836               0.930818   
1703         55.599998  0.762675    -0.038384               0.844209   

      afecto_positivo  afecto_negativo  confianza  calidad_democracia  \
1            0.583926         0.237092   0.611545           -2.044093   
10           0.424125         0.404904   0.364666                 NaN   
12           0.640024         0.279257        NaN            0.048114   
21           0.713300         0.318997   0.435338                 NaN   
28           0.591043         0.292946        NaN                 NaN   
...               ...              ...        ...                 ...   
1678         0.461114         0.314870   0.308151                 NaN   
1682         0.727505         0.122659   0.413418            0.133997   
1690         0.702698         0.350963   0.606715                 NaN   
1694         0.735503         0.218419   0.285287           -1.353181   
1703         0.710119         0.211726   0.550508                 NaN   

      calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
1           -1.635025               1.722688                 NaN   
10                NaN               1.408344                 NaN   
12          -0.264635               1.919640                 NaN   
21                NaN               2.640531                 NaN   
28                NaN               1.973943                 NaN   
...               ...                    ...                 ...   
1678              NaN               2.402008                 NaN   
1682        -0.567933               1.889908                 NaN   
1690              NaN               2.783419                 NaN   
1694        -1.717821               2.024098                 NaN   
1703              NaN               2.498696                 NaN   

      gini_banco_mundial_promedio  
1                             NaN  
10                            NaN  
12                        0.30325  
21                        0.30325  
28                        0.27600  
...                           ...  
1678                      0.35700  
1682                      0.52740  
1690                      0.52740  
1694                      0.43200  
1703                      0.43200  

[250 rows x 17 columns]

vemos como el data set de una dimensión de

In [14]:
felicidad.shape

(1704, 17)

se redujo a 

In [16]:
felicidad_anio.shape

(250, 17)

Ahora, realicemos el Análisis Descriptivo de estos datos

In [20]:
felicidad_anio.describe()

anio  escalera_vida     log_pib  soporte_social  \
count   250.000000     250.000000  240.000000      249.000000   
mean   2013.896000       5.481845    9.203628        0.814416   
std       4.491534       1.068550    1.182632        0.113452   
min    2009.000000       2.694303    6.457201        0.325693   
25%    2009.000000       4.686966    8.270360        0.754646   
50%    2018.000000       5.390879    9.382841        0.834716   
75%    2018.000000       6.276394   10.132141        0.904390   
max    2018.000000       7.858107   11.670484        0.984489   

       expectativa_vida    libertad  generosidad  percepcion_corrupcion  \
count        245.000000  248.000000   239.000000             240.000000   
mean          63.691978    0.740683    -0.018916               0.746372   
std            7.157827    0.142678     0.159093               0.189907   
min           44.619999    0.257534    -0.336385               0.035198   
25%           59.000000    0.648587    -0.129488               0.689930   
50%           65.160004    0.760047    -0.030179               0.801145   
75%           68.500000    0.858693     0.069399               0.866134   
max           76.800003    0.969898     0.518503               0.978800   

       afecto_positivo  afecto_negativo   confianza  calidad_democracia  \
count       248.000000       248.000000  217.000000          113.000000   
mean          0.708018         0.272400    0.496210           -0.234035   
std           0.110694         0.086707    0.202993            0.846086   
min           0.424125         0.092696    0.079710           -2.167385   
25%           0.611759         0.206347    0.343467           -0.882276   
50%           0.734457         0.261765    0.476145           -0.331738   
75%           0.793756         0.326980    0.619251            0.412251   
max           0.897641         0.543836    0.988120            1.502807   

       calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
count       113.000000             250.000000           53.000000   
mean         -0.059646               2.115297            0.376698   
std           0.938551               0.476345            0.083572   
min          -1.717821               1.198477            0.248000   
25%          -0.749496               1.753263            0.314000   
50%          -0.237261               2.040677            0.354000   
75%           0.544111               2.361383            0.442000   
max           2.121312               3.718958            0.545000   

       gini_banco_mundial_promedio  
count                   222.000000  
mean                      0.386281  
std                       0.081774  
min                       0.211000  
25%                       0.325715  
50%                       0.369875  
75%                       0.432150  
max                       0.626000

Recordemos que la felicidad o también conocida como Escalera de Vida indica la percepción de calidad de vida en una escala de 0 a 10.  Mientras más alto el puntaje, mayor felicidad. <br>
1. Como vemos el promedio de felicidad en nuestra muestra es de 5.48, es decir una felicidad media, ya que la escala es de 0 a 10. La desviación estandar es de 1.07 lo que significa que la mayoría de los puntajes de felicidad se encuentran dentro del intervalo $[\mu-2\sigma,\mu+2\sigma] = [3.34,7.62]$, lo cual indica que la mayoría de los paises de esta muestra tienen niveles de felicidad relativamente cercanos al promedio. <br>
2. Vemos también que el mínimo puntaje de felicidad es casi 3 puntos y el máximo es casi 8 puntos. Esto significa que hay variación en la percepción de calidad de vida entre los paises, teniendo algunos niveles bastantes bajos y otros muy altos.
3. Respecto a los rangos intercuartílicos que proporcionan información sobre la dispersión de los datos:
   - El 25% de la muestra tiene un valor de felicidad menor o igual a 4.68 puntos.
   - La mediana, que representa el 50% de la muestra, tiene un valor de felicidad menor o igual a 5.39 puntos, lo que indica que la mitad de los paises en nuestra muestra tienen un nivel de felicidad por debajo de este valor.
   - El 75% de la muestra tiene a lo más valor de 6.28 puntos.

En resumen, los paises en nuestra muestra tienen niveles de felicidad cercanos al promedio, pero existen diferencias significativas en paises, con algunos experimentando niveles de felicidad más bajos y otros más alto.

## ¿Cómo acceden a la información de la felicidad del primer y último año para los dos países de su elección?

Veamos que paises existen en la muestra filtrada por el primer y ultimo año

In [27]:
felicidad_anio['pais'].unique()

array(['Afghanistán', 'Albania', 'Algeria', 'Argentina', 'Armenia',
       'Australia', 'Austria', 'Azerbaijan', 'Bahrain', 'Bangladesh',
       'Bielorrusia', 'Bélgica', 'Benin', 'Bolivia',
       'Bosnia y Herzegovina', 'Botswana', 'Brasil', 'Bulgaria',
       'Burkina Faso', 'Burundi', 'Cambodia', 'Camerún', 'Canadá', 'Chad',
       'Chile', 'China', 'Colombia', 'Comoros', 'Congo (Brazzaville)',
       'Congo (Kinshasa)', 'Costa Rica', 'Croacia', 'Chipre',
       'República Checa', 'Dinamarca', 'Djibouti', 'República Dominicana',
       'Ecuador', 'Egipto', 'El Salvador', 'Estonia', 'Etiopía',
       'Finlandia', 'Francia', 'Gabón', 'Gambia', 'Georgia', 'Alemania',
       'Ghana', 'Greece', 'Guatemala', 'Guinea', 'Haití', 'Honduras',
       'Hong Kong S.A.R. of China', 'Hungría', 'India', 'Indonesia',
       'Iran', 'Irak', 'Irlanda', 'Israel', 'Italia', 'Costa de Marfil',
       'Japón', 'Jordania', 'Kazakhstan', 'Kenia', 'Kosovo', 'Kuwait',
       'Kyrgyzstan', 'Laos', 'Latvia', '

eligiremos `El Salvador` y `Ecuador`

In [29]:
felicidad_dos = felicidad_anio[(felicidad_anio['pais']=='Ecuador') | (felicidad_anio['pais']=='El Salvador')] 
felicidad_dos

pais  anio  escalera_vida   log_pib  soporte_social  \
430      Ecuador  2009       6.021803  9.125170        0.779398   
439      Ecuador  2018       6.128010  9.274455        0.851345   
456  El Salvador  2009       6.839087  8.732031        0.734113   
465  El Salvador  2018       6.276246  8.911959        0.823999   

     expectativa_vida  libertad  generosidad  percepcion_corrupcion  \
430         66.620003  0.736881    -0.111119               0.774305   
439         68.500000  0.869364    -0.104021               0.830743   
456         63.880001  0.670932    -0.101633               0.647528   
465         66.099998  0.863207    -0.096787               0.826600   

     afecto_positivo  afecto_negativo  confianza  calidad_democracia  \
430         0.840363         0.255968   0.535495           -0.461775   
439         0.876475         0.328171   0.402085                 NaN   
456         0.850378         0.243231   0.585729            0.025729   
465         0.871536         0.265371   0.323992                 NaN   

     calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
430        -1.036553               2.232644               0.484   
439              NaN               2.485788                 NaN   
456        -0.180710               1.934376               0.458   
465              NaN               2.516374                 NaN   

     gini_banco_mundial_promedio  
430                     0.496000  
439                     0.496000  
456                     0.457824  
465                     0.457824

In [30]:
felicidad_dos.describe()

anio  escalera_vida   log_pib  soporte_social  expectativa_vida  \
count     4.000000       4.000000  4.000000        4.000000          4.000000   
mean   2013.500000       6.316287  9.010904        0.797214         66.275001   
std       5.196152       0.363819  0.238101        0.051468          1.900552   
min    2009.000000       6.021803  8.732031        0.734113         63.880001   
25%    2009.000000       6.101459  8.866977        0.768077         65.544999   
50%    2013.500000       6.202128  9.018564        0.801699         66.360001   
75%    2018.000000       6.416956  9.162491        0.830836         67.090002   
max    2018.000000       6.839087  9.274455        0.851345         68.500000   

       libertad  generosidad  percepcion_corrupcion  afecto_positivo  \
count  4.000000     4.000000               4.000000         4.000000   
mean   0.785096    -0.103390               0.769794         0.859688   
std    0.097571     0.005967               0.085462         0.017150   
min    0.670932    -0.111119               0.647528         0.840363   
25%    0.720394    -0.105796               0.742610         0.847874   
50%    0.800044    -0.102827               0.800452         0.860957   
75%    0.864746    -0.100422               0.827636         0.872771   
max    0.869364    -0.096787               0.830743         0.876475   

       afecto_negativo  confianza  calidad_democracia  calidad_entrega  \
count         4.000000   4.000000            2.000000         2.000000   
mean          0.273185   0.461825           -0.218023        -0.608631   
std           0.037763   0.120203            0.344717         0.605172   
min           0.243231   0.323992           -0.461775        -1.036553   
25%           0.252784   0.382562           -0.339899        -0.822592   
50%           0.260669   0.468790           -0.218023        -0.608631   
75%           0.281071   0.548054           -0.096147        -0.394671   
max           0.328171   0.585729            0.025729        -0.180710   

       de_escalera_pais_anio  gini_banco_mundial  gini_banco_mundial_promedio  
count               4.000000            2.000000                     4.000000  
mean                2.292295            0.471000                     0.476912  
std                 0.270380            0.018385                     0.022041  
min                 1.934376            0.458000                     0.457824  
25%                 2.158077            0.464500                     0.457824  
50%                 2.359216            0.471000                     0.476912  
75%                 2.493434            0.477500                     0.496000  
max                 2.516374            0.484000                     0.496000

como vemos la media en términos de felicidad de estos paises está en 6.32, es decir, mayor al promedio 5.48 de felicidad a nivel mundial de la muestra `felicidad_anio` y ligeramente mayor al 75% de la muestra cuyo valor era de 6.28 puntos. Por tanto, estos dos paises a elección sí son más felices en estos años que el resto de paises en los mismos años, 2009 y 2018.

## Opcional: Si quieren pueden mostrar cómo revisan otras columnas que expliquen por qué los países que eligieron son más o menos felices que el promedio global. Por ejemplo, el país X tiene mejor soporte social que el promedio del mundo y por eso es más feliz.

Realicemos el análisis por los dos paises.

In [37]:
felicidad_Ecuador = felicidad_dos[felicidad_dos['pais']=='Ecuador']

In [38]:
felicidad_Salvador = felicidad_dos[felicidad_dos['pais']=='El Salvador']

obtenemos su respectivo Análisis Descriptivo

In [40]:
felicidad_Ecuador.describe()

anio  escalera_vida   log_pib  soporte_social  expectativa_vida  \
count     2.000000       2.000000  2.000000        2.000000          2.000000   
mean   2013.500000       6.074907  9.199812        0.815371         67.560001   
std       6.363961       0.075100  0.105561        0.050874          1.329359   
min    2009.000000       6.021803  9.125170        0.779398         66.620003   
25%    2011.250000       6.048355  9.162491        0.797385         67.090002   
50%    2013.500000       6.074907  9.199812        0.815371         67.560001   
75%    2015.750000       6.101459  9.237134        0.833358         68.030001   
max    2018.000000       6.128010  9.274455        0.851345         68.500000   

       libertad  generosidad  percepcion_corrupcion  afecto_positivo  \
count  2.000000     2.000000               2.000000         2.000000   
mean   0.803122    -0.107570               0.802524         0.858419   
std    0.093680     0.005019               0.039908         0.025535   
min    0.736881    -0.111119               0.774305         0.840363   
25%    0.770002    -0.109345               0.788414         0.849391   
50%    0.803122    -0.107570               0.802524         0.858419   
75%    0.836243    -0.105796               0.816634         0.867447   
max    0.869364    -0.104021               0.830743         0.876475   

       afecto_negativo  confianza  calidad_democracia  calidad_entrega  \
count         2.000000   2.000000            1.000000         1.000000   
mean          0.292069   0.468790           -0.461775        -1.036553   
std           0.051055   0.094335                 NaN              NaN   
min           0.255968   0.402085           -0.461775        -1.036553   
25%           0.274019   0.435438           -0.461775        -1.036553   
50%           0.292069   0.468790           -0.461775        -1.036553   
75%           0.310120   0.502143           -0.461775        -1.036553   
max           0.328171   0.535495           -0.461775        -1.036553   

       de_escalera_pais_anio  gini_banco_mundial  gini_banco_mundial_promedio  
count               2.000000               1.000                        2.000  
mean                2.359216               0.484                        0.496  
std                 0.179000                 NaN                        0.000  
min                 2.232644               0.484                        0.496  
25%                 2.295930               0.484                        0.496  
50%                 2.359216               0.484                        0.496  
75%                 2.422502               0.484                        0.496  
max                 2.485788               0.484                        0.496

In [41]:
felicidad_Salvador.describe()

anio  escalera_vida   log_pib  soporte_social  expectativa_vida  \
count     2.000000       2.000000  2.000000        2.000000          2.000000   
mean   2013.500000       6.557667  8.821995        0.779056         64.990000   
std       6.363961       0.397989  0.127228        0.063559          1.569775   
min    2009.000000       6.276246  8.732031        0.734113         63.880001   
25%    2011.250000       6.416956  8.777013        0.756584         64.435000   
50%    2013.500000       6.557667  8.821995        0.779056         64.990000   
75%    2015.750000       6.698377  8.866977        0.801528         65.544999   
max    2018.000000       6.839087  8.911959        0.823999         66.099998   

       libertad  generosidad  percepcion_corrupcion  afecto_positivo  \
count  2.000000     2.000000               2.000000         2.000000   
mean   0.767070    -0.099210               0.737064         0.860957   
std    0.135959     0.003427               0.126623         0.014962   
min    0.670932    -0.101633               0.647528         0.850378   
25%    0.719001    -0.100422               0.692296         0.855667   
50%    0.767070    -0.099210               0.737064         0.860957   
75%    0.815139    -0.097998               0.781832         0.866247   
max    0.863207    -0.096787               0.826600         0.871536   

       afecto_negativo  confianza  calidad_democracia  calidad_entrega  \
count         2.000000   2.000000            1.000000          1.00000   
mean          0.254301   0.454860            0.025729         -0.18071   
std           0.015655   0.185076                 NaN              NaN   
min           0.243231   0.323992            0.025729         -0.18071   
25%           0.248766   0.389426            0.025729         -0.18071   
50%           0.254301   0.454860            0.025729         -0.18071   
75%           0.259836   0.520295            0.025729         -0.18071   
max           0.265371   0.585729            0.025729         -0.18071   

       de_escalera_pais_anio  gini_banco_mundial  gini_banco_mundial_promedio  
count               2.000000               1.000                     2.000000  
mean                2.225375               0.458                     0.457824  
std                 0.411535                 NaN                     0.000000  
min                 1.934376               0.458                     0.457824  
25%                 2.079875               0.458                     0.457824  
50%                 2.225375               0.458                     0.457824  
75%                 2.370874               0.458                     0.457824  
max                 2.516374               0.458                     0.457824

recordemos el análisis descriptivo a nivel mundial de los años 2009 y 2018

In [43]:
felicidad_anio.describe()

anio  escalera_vida     log_pib  soporte_social  \
count   250.000000     250.000000  240.000000      249.000000   
mean   2013.896000       5.481845    9.203628        0.814416   
std       4.491534       1.068550    1.182632        0.113452   
min    2009.000000       2.694303    6.457201        0.325693   
25%    2009.000000       4.686966    8.270360        0.754646   
50%    2018.000000       5.390879    9.382841        0.834716   
75%    2018.000000       6.276394   10.132141        0.904390   
max    2018.000000       7.858107   11.670484        0.984489   

       expectativa_vida    libertad  generosidad  percepcion_corrupcion  \
count        245.000000  248.000000   239.000000             240.000000   
mean          63.691978    0.740683    -0.018916               0.746372   
std            7.157827    0.142678     0.159093               0.189907   
min           44.619999    0.257534    -0.336385               0.035198   
25%           59.000000    0.648587    -0.129488               0.689930   
50%           65.160004    0.760047    -0.030179               0.801145   
75%           68.500000    0.858693     0.069399               0.866134   
max           76.800003    0.969898     0.518503               0.978800   

       afecto_positivo  afecto_negativo   confianza  calidad_democracia  \
count       248.000000       248.000000  217.000000          113.000000   
mean          0.708018         0.272400    0.496210           -0.234035   
std           0.110694         0.086707    0.202993            0.846086   
min           0.424125         0.092696    0.079710           -2.167385   
25%           0.611759         0.206347    0.343467           -0.882276   
50%           0.734457         0.261765    0.476145           -0.331738   
75%           0.793756         0.326980    0.619251            0.412251   
max           0.897641         0.543836    0.988120            1.502807   

       calidad_entrega  de_escalera_pais_anio  gini_banco_mundial  \
count       113.000000             250.000000           53.000000   
mean         -0.059646               2.115297            0.376698   
std           0.938551               0.476345            0.083572   
min          -1.717821               1.198477            0.248000   
25%          -0.749496               1.753263            0.314000   
50%          -0.237261               2.040677            0.354000   
75%           0.544111               2.361383            0.442000   
max           2.121312               3.718958            0.545000   

       gini_banco_mundial_promedio  
count                   222.000000  
mean                      0.386281  
std                       0.081774  
min                       0.211000  
25%                       0.325715  
50%                       0.369875  
75%                       0.432150  
max                       0.626000

### Ecuador

Vemos que Ecuador tiene mejor puntuación en expectativa_vida, libertad, afecto_positivo que el promedio mundial de los otros paises. A pesar de no tener un mejor promedio en generosidad, percepcion_corrupcion, afecto_negativo, confianza, calidad_democracia, calidad_entrega, o en gini_banco_mundial y gini_banco_mundial_promedio estos dos último en términos de igualdad económica.

### El Salvador

Vemos que El Salvador tiene mejor puntuación en  expectativa_vida, libertad, afecto_negativo, afecto_positivo, calidad_democracia que el promedio mundial de los otros paises. A pesar de no tener un mejor promedio en confianza, calidad_entrega, gini_banco_mundial, gini_banco_mundial_promedio que el resto de paises.